tmt/steps/report/junit.py

```diff
@@ -49,19 +62,20 @@ class ReportJUnit(tmt.steps.report.ReportPlugin):
     _keys = ["file"]
 
     @classmethod
-    def options(cls, how=None):
+    def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecoratorType]:
         """ Prepare command line options for connect """
-        return [
+        return cast(List[tmt.options.ClickOptionDecoratorType], [
```

Comment:
We should try to get rid of this `cast()`. PR #1324 has [a proposal](https://github.com/teemtee/tmt/pull/1324#discussion_r926567419) on how to do that, so I'll wait if it gets approved before changing it here also.

Reply:
I will update this in the same way as for the `html` report.